docs/developer-reference/contracts/deployment-addresses/mainnet.md

```diff
@@ -1,60 +1,47 @@
 # Mainnet Deployment Addresses
 
-## Pool Factories
+## Active Contracts
 
-| Name                | Address                                    |
-| ------------------- | ------------------------------------------ |
-| StablePoolFactory   | 0xB9d01CA61b9C181dA1051bFDd28e1097e920AB14 |
-| MockStablePool      | 0x89Ef89Fd9a6ec73bcE588F309C1F65C406d2891C |
-| WeightedPoolFactory | 0x201efd508c8DfE9DE1a13c2452863A78CB2a86Cc |
-| MockWeightedPool    | 0x527d0E14acc53FB040DeBeae1cAb973D23FB3568 |
+::: info More Details
+For more information on specific deployments as well as changelogs for different contract versions, please see the [deployment tasks](https://github.com/balancer/balancer-deployments/tree/master/v3/tasks).
```

Comment:
I replaced the old link to now point to the v3 deployment tasks

Reply:
Oh thanks a lot! I missed that